### DISCUSSION TITLE
How to go to parent directory in the file picker?

### DISCUSSION THREAD

**Discussion by wmstack** [helix-editor/helix#2496] [Q&A]
I use <kbd>space</kbd>+<kbd>f</kbd> from normal mode to open the file picker. How do I navigate to the parent directory from the file picker?

**Comment by schell** [comment]
I think there may be a bug because when I use `space`+`F` I can only navigate within the current project. This happens even if I open another terminal window.

**Reply by the-mikedavis** [reply]
That's unrelated to this discussion.

Space-F opens the picker in the current directory and you can check which directory that is with `:pwd`. Helix tries to find the workspace when you open a file in a project (using either the `roots` keys from `languages.toml` or the `.git` directory, whichever comes first). You can open helix on a directory with `hx path/to/directory` to set the CWD like that, or use the `:cd` command if the editor is already opened.
